### PR TITLE
Add `NEW tBTC dApp` modal

### PR DESCRIPTION
**Pull request by r-czajkowski**
~Depends on: #335~
Closes: #344 

This modal should be displayed only once and only on the `Bridge` page. To determine if the user has seen this modal before, we use local storage.

### CODE REVIEW

File: src/components/tBTC/TakeNoteList.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+export const TakeNoteList: FC<{ size?: "sm" | "md" }> = ({ size = "md" }) => {
+  const LabelComponent = size === "sm" ? LabelSm : LabelMd
+  const BodyComponent = size === "sm" ? BodySm : BodyMd
+  return (
+    <List spacing="4">
+      <ListItem>
+        <LabelComponent>minting duration</LabelComponent>
+        <List spacing="2">
```
Comment (michalsmiarowski) [inline src/components/tBTC/TakeNoteList.tsx:36]: O would add a margin-top here so it's more similar to figma designs.

Figma designs:
image

Cuttent implementation:
image

File: src/components/tBTC/TakeNoteList.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                ~1 to 3 hours
+              </Box>
+              .
+            </BodyComponent>
+          </ListItemWithIcon>
+        </List>
+      </ListItem>
+      <ListItem>
+        <LabelComponent>funds time lock</LabelComponent>
+        <List spacing="2">
```
Comment (michalsmiarowski) [inline src/components/tBTC/TakeNoteList.tsx:50]: Same as above if you agree

File: src/components/tBTC/TakeNoteList.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                9 months
+              </Box>
+              .
+            </BodyComponent>
+          </ListItemWithIcon>
+        </List>
+      </ListItem>
+      <ListItem>
+        <LabelComponent>unminting tbtc</LabelComponent>
+        <List spacing="2">
```
Comment (michalsmiarowski) [inline src/components/tBTC/TakeNoteList.tsx:64]: Same as above if you agree

File: src/components/Modal/tBTC/NewTBTCApp.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    <>
+      <ModalHeader>Take note</ModalHeader>
+      <ModalCloseButton />
+      <ModalBody>
+        <Image
+          src={tbtcAppBannerIllustration}
+          maxH={{ base: "140px", xl: "unset" }}
+          mx="auto"
+          mb="14"
+        />
```
Comment (michalsmiarowski) [inline src/components/Modal/tBTC/NewTBTCApp.tsx:34]: On the Figma we also have a checkered pattern as a background of the image:

image

Current implementation:
image

File: src/components/Modal/tBTC/NewTBTCApp.tsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        <BodyLg mb="12" color="gray.500">
+          Take note of the following before you proceed.
+        </BodyLg>
+        <Flex justifyContent="center">
+          <TakeNoteList size="sm" />
+        </Flex>
+        <BodySm mt="4.5rem" textAlign="center" color="gray.500">
+          By clicking the button below, you agree with the terms and you are
+          aware of the results.
+        </BodySm>
```
Comment (michalsmiarowski) [inline src/components/Modal/tBTC/NewTBTCApp.tsx:45]: We could add some margin/padding on the sides here too so it's more similar to figma designs

**Review by michalsmiarowski — CHANGES_REQUESTED** [review]
Looking good! Left some comments to look at before the merge

File: src/components/tBTC/TakeNoteList.tsx (hunk 1 above)
Comment (r-czajkowski) [reply src/components/tBTC/TakeNoteList.tsx:36]: a0c826d

File: src/components/tBTC/TakeNoteList.tsx (hunk 2 above)
Comment (r-czajkowski) [reply src/components/tBTC/TakeNoteList.tsx:50]: a0c826d

File: src/components/tBTC/TakeNoteList.tsx (hunk 3 above)
Comment (r-czajkowski) [reply src/components/tBTC/TakeNoteList.tsx:64]: a0c826d

File: src/components/Modal/tBTC/NewTBTCApp.tsx (hunk 5 above)
Comment (r-czajkowski) [reply src/components/Modal/tBTC/NewTBTCApp.tsx]: a0c826d

**Review by michalsmiarowski — APPROVED** [review]
LGTM 🔥

**Pull request merged**